ios/SalesforceReactTests/Podfile

```diff
@@ -0,0 +1,48 @@
+platform :ios, '11.0'
+
+project 'SalesforceReactTestApp.xcodeproj'
+target 'SalesforceReactTestApp' do
+end
+
+target 'SalesforceReactTests' do
+
+source 'https://github.com/CocoaPods/Specs.git'
+
+pod 'yoga', :path => './node_modules/react-native/ReactCommon/yoga'
+pod 'DoubleConversion', :podspec => './node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
+pod 'glog', :podspec => './node_modules/react-native/third-party-podspecs/glog.podspec'
+pod 'Folly', :podspec => './node_modules/react-native/third-party-podspecs/Folly.podspec'
+pod 'React', :path => './node_modules/react-native', :subspecs => [
+    'DevSupport',
+    'jschelpers',
+    'cxxreact',
+    'CxxBridge',
+    'ART',
+    'RCTActionSheet',
+    'RCTAnimation',
+    'RCTCameraRoll',
+    'RCTGeolocation',
+    'RCTImage',
+    'RCTNetwork',
+    'RCTPushNotification',
+    'RCTSettings',
+    'RCTText',
+    'RCTVibration',
+    'RCTWebSocket',
+    'RCTLinkingIOS',
+    'RCTTest'
+]
+
+pod 'SalesforceAnalytics', :path => './mobile_sdk/SalesforceMobileSDK-iOS'
```

Comment:
Do we need these pods? Wouldn't they get pulled in through `SalesforceReact` transitively?

Reply:
That's what we do in all our template apps.We could try without - I think we might have run into issues in the past.

Reply:
See other comment. I don't think we need to call out transitive dependencies.

Reply:
Actually we need to list the ones we want to pull from alternative locations (meaning location given by a :path or :git), otherwise it will look for the dependency in the source specs.

Reply:
Gotcha. That makes sense.